docs/rules/no-observers.md

```diff
@@ -2,7 +2,7 @@
 
 ### Rule name: `no-observers`
 
-Usage of observers is very easy **BUT** it leads to hard to reason about consequences. Unless observers are necessary, it's better to avoid them.
+Observers deal with data changes contrary to Ember's Data Down, Actions Up (DDAU) convention. They also are synchronous by default which [has problems](https://emberjs.github.io/rfcs/0494-async-observers.html#motivation). You can usually state management problems using other robust tools in Ember's toolbox such as actions, computed properties, or component life cycle hooks. Observers do have some limited uses. They let you reflect state from your application to foreign interfaces that don't follow Ember's data flow conventions. For the canonical, in-depth reasoning to why not see [@ef4's](https://github.com/ef4/) canonical answer [here](https://discuss.emberjs.com/t/why-should-i-not-use-observers-in-my-ember-application/16868/3)
```

Comment:
Can you split this into a few paragraphs and bullet points to make it easier to read?

Also `You can usually state management problems` looks like it has a missing word.